src/gamestate/commands.cpp

```diff
@@ -3710,6 +3710,28 @@ void execute_toggle_immigrator_province(sys::state& state, dcon::nation_id sourc
 	sys::toggle_modifier_from_province(state, prov, state.economy_definitions.immigrator_modifier, sys::date{});
 }
 
+void release_subject(sys::state& state, dcon::nation_id source, dcon::nation_id target) {
+	payload p;
+	memset(&p, 0, sizeof(payload));
+	p.type = command_type::release_subject;
+	p.source = source;
+	p.data.diplo_action.target = target;
+	add_to_command_queue(state, p);
+}
+bool can_release_subject(sys::state& state, dcon::nation_id source, dcon::nation_id target) {
+	auto ovr = state.world.nation_get_overlord_as_subject(target);
+	if(!bool(ovr))
+		return false;
+	if(state.world.overlord_get_ruler(ovr) != source)
```

Comment:
because without having an overload, it would return the invalid handle here, which will not be equal to source